app/javascript/api/invoices.js

```diff
@@ -56,4 +56,13 @@ export default {
       url: path,
     });
   },
+  notifyUser(user_name, user_id) {
```

Comment:
Identifier 'user_name' is not in camel case     camelcase<br>Identifier 'user_id' is not in camel case       camelcase